src/modules/tasks/task-list/tasks.html

```diff
@@ -86,7 +86,10 @@ <h1 class="bold-text">Tasks</h1>
                     <div class="bold-text ">Submitted</div>
                     <div class="pull-left ">{{task.created_at| date: 'dd MMM yyyy HH:mm:ss'}}</div>
                 </div>
-                <div class="ft-column bold-text status-col ">{{taskCntrl.getStatusText(task.status)}}</div>
+                <div class="ft-column status-col ">
+                <div class="bold-text ">{{taskCntrl.getStatusText(task.status)}}</div>
+                    <div class="pull-left ">{{task.created_at| date: 'dd MMM yyyy HH:mm:ss'}}</div>
```

Comment:
@a2batic It is not properly indented and you need  map updated_at property  not created_at.

Reply:
Acknowledge